### DISCUSSION TITLE
CoPilot chokes on larger suggestions.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#61008] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I am working on a library for hex grids in games. This is pretty well known code, I'm just creating my own implementation for educational purposes and because I prefer writing my own over the existing libraries.

So when I am creating a unit test for a method that returns rings of hexes, CoPilot suggested this:

![image](https://github.com/community/community/assets/30846057/189f43a7-c02b-47fe-96be-4ac6928417e8)

Note the last line is incomplete; as well I believe the values are wrong (I haven't checked them, but the values it proposed for a ring count of 2 were completely off past the tenth item.

It's been great so far, and saves me a lot of typing, but occasionally, on larger bits of code, it chokes.

Thanks,

Morgan

**Comment by jshtuazon2** [comment]
#61008
